### PR TITLE
Update Viperize to use new List Semantics from pflags.

### PR DESCRIPTION
**Pull request by therealmitchconnors**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -118,6 +118,8 @@ github.com/spf13/jwalterweatherman v1.0.0 h1:XHEdyB+EcvlqZamSM4ZOMGlc93t6AcsBEu9
 github.com/spf13/jwalterweatherman v1.0.0/go.mod h1:cQK4TGJAtQXfYWX+Ddv3mKDzgVb68N+wFjFa4jdeBTo=
 github.com/spf13/pflag v1.0.3 h1:zPAT6CGy6wXeQ7NtTnaTerfKOsV6V6F8agHXFiazDkg=
 github.com/spf13/pflag v1.0.3/go.mod h1:DYY7MBk1bdzusC3SYhjObp+wFpr4gzcvqqNjLnInEg4=
+github.com/spf13/pflag v1.0.4 h1:iDFxJFdWvBR5ieANJFvx+fxUFC0lheY/+K/lV73QbEs=
+github.com/spf13/pflag v1.0.4/go.mod h1:5F7vF+0t+QPgqofPFTZZG2xnxtA/FGYmPu6WbdsXIvo=
```
Comment (howardjohn) [inline go.sum]: nit: probably could run `go mod tidy`

File: CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1 @@
-*            @geeknoid @ozevren
-/appsignals  @louiscryan
-/filewatcher @lichuqiang
+* @istio/wg-test-and-release-maintainers
```
Comment (geeknoid) [inline CODEOWNERS]: Hmm. This is old, please rebase the PR.

File: go.sum (hunk 1 above)
Comment (therealmitchconnors) [reply go.sum]: this is after having run go mod tidy... I am not sure why there are two versions listed here... perhaps an indirect dependency?

File: CODEOWNERS (hunk 2 above)
Comment (therealmitchconnors) [reply CODEOWNERS]: This is from the most recent commit on master (https://github.com/istio/pkg/commit/f287e58fcc78fab3757657fe4eaeb0885ea18882).  I'm not sure why it's showing up as part of my PR...

**Comment by geeknoid** [comment]
Needs https://github.com/istio/test-infra/pull/1823 and #68

**Comment by geeknoid** [comment]
Please rebase, that should help with the CI problems.

**Pull request merged**